README.md

```diff
@@ -1,101 +1,134 @@
+
+### Error Handling
 
-### Error Handling Recommendations
 - Your code shall raise an exception when asked to reserve a room that is not available
 
 ## Wave Three: Blocks of Rooms
 
 If you are not familiar with what a block of hotel rooms, here is a brief description:
 
-> A Block Booking refers to a group of rooms set aside for a specific customer -- usually for a set period of time.
+> A Block Booking refers to a group of rooms set aside for a specific group of customers for a set period of time.
 >
-> Room blocks are commonly reserved for conventions and meetings or groups in general. A room block is usually under a firm agreement and is for a set period of time. Rooms blocked are not available for sales to other customers.
+> Room blocks are commonly created for large events like weddings or conventions. A number of rooms are set aside, and are made available for reservation by certain customers at a discounted rate. These rooms are not available to be reserved by the general public.
 
 ### User Stories
-- As an administrator, I should be able to create a new block of rooms
-  - To create a block you need a date range, collection of rooms and a discounted room rate
-  - The collection of rooms should only include rooms that are available for the given date range
-- As an administrator, I should be able to check whether a given block has any rooms available
-- As an administrator, I should be able to reserve a room from within a block of rooms
 
-### Assumptions
-- Assume that a block can be created for a maximum of 5 rooms
-- Assume that when a room is reserved from a block of rooms, it will **always** match the date range of the block
-- All of the logic from Wave 2 should still work, i.e. individual reservations
+- As an administrator, I can create a block of rooms
+    - To create a block you need a date range, collection of rooms and a discounted room rate
+    - The collection of rooms should only include rooms that are available for the given date range
+    - If a room is set aside in a block, it is not available for reservation by the general public, nor can it be included in another block
+- As an administrator, I can check whether a given block has any rooms available
+- As an administrator, I can reserve a room from within a block of rooms
+
+### Constraints
 
-### Error Handling Recommendations
-- If a room is reserved in a block, it is not available for reservation by the general public
+- A block can contain a maximum of 5 rooms
+- When a room is reserved from a block of rooms, the reservation dates will always match the date range of the block
+- All of the availability checking logic from Wave 2 should now respect room blocks as well as individual reservations
+
+## Optional Enhancements
 
-## Optional Requirements
 ### User Stories
-- As an administrator, I should be able to set different rates for different rooms
+
+- As an administrator, I can set different rates for different rooms
```

Comment:
just so I'm clear, the structure of this segment is:
- Optional Enhancements
  - One optional enhancement is to implement the feature described by this user story
  - The other two optional enhancements are technical, and aren't related to this user story

?

Reply:
Correct, there are three recommendations for things they could work on. One is a user story, the other two are more backend / technical things.

Reply:
I'll see if I can make that more clear